### PR TITLE
Use new baggage query 🛄 👜 💼 🎒 

### PR DESCRIPTION
**Pull request by kantord**
This reverts commit 9de0fa78645d62c8b34aceaa85fa1e8920aa970f.

To test in local, you need to run your own GrahpQL in local, with this branch checked out: 
https://github.com/kiwicom/graphql/pull/273

Then you'll have to edit `.env` to make sure your local SmartFAQ uses your local GraphQL.

Not mergeable until the pull request is merged in GraphQL.

<br/><br/><br/><url>LiveURL: https://smartfaq-re-enable-baggage.surge.sh</url>

### CODE REVIEW

**Comment by jaroslav-kubicek** [comment]
conflicts, cc @kantord

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js [hunk 1]
```diff
@@ -89,11 +105,16 @@ export const BaggageDescription = ({
 export default createFragmentContainer(
   BaggageDescription,
   graphql`
-    fragment BaggageDescription on Baggage {
-      height
-      weight
-      width
-      length
+    fragment BaggageDescription on BookingBaggage {
```
Comment (jaroslav-kubicek) [inline src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js]: I would prefer to define prop name with `_propName` since we use it elsewhere

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/__test__/BaggageDescription.test.js [hunk 2]
```diff
@@ -8,15 +8,18 @@ import { BaggageDescription } from '../BaggageDescription';
...
-  weight: 10,
+  bag: {
+    height: 40,
+    width: 30,
+    length: 15,
+    weight: 10,
+    category: 'CHECKED',
+    note: 'asd',
+  },
+  quantity: 0,
```
Comment (jaroslav-kubicek) [inline src/StaticFAQ/FAQExtraInfo/BaggageInfo/__test__/BaggageDescription.test.js]: will the GraphQL return bag if quantity is 0?

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js [hunk 3]
```diff
@@ -81,13 +81,13 @@ class BaggageInfo extends React.Component<Props> {
...
         <div className="iconTitle">
-          <Baggages customColor="black" />
+          <BaggageChecked customColor="black" />
         </div>
         <h1 className="title">Your baggage</h1>
         <div className="subtitle">
           <Text type="attention">Here you can see your baggage allowance.</Text>
         </div>
-        <BaggageSummary data={baggage} mmbUrl={directAccessURL} />
+        <BaggageSummary data={baggage || null} mmbUrl={directAccessURL} />
```
Comment (jaroslav-kubicek) [inline src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js]: if no baggage is allowed - a list is empty, nothing is rendered if I'm correct. I would add some note with italic (as it's here https://zpl.io/aN09rwe for "You cannot add more baggage to this booking.") saying you don't have any baggage allowance

Comment (jaroslav-kubicek) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js]: + would be nice to add to some unit tests for this, covering at least these two cases - 1) you have baggage 2) you don't have any baggage

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/__test__/BaggageDescription.test.js (hunk 2 above)
Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/__test__/BaggageDescription.test.js]: Good question. GraphQL currently relies on the API to handle this edge case. I believe the API has to handle this, but to be safe maybe we could just handle it on GraphQL too.

**Comment by kantord** [comment]
:heavy_check_mark: conflicts resolved

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js (hunk 3 above)
Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js]: > if no baggage is allowed - a list is empty, nothing is rendered if I'm correct. I would add some note with italic (as it's here https://zpl.io/aN09rwe for "You cannot add more baggage to this booking.") saying you don't have any baggage allowance

Good idea, however, is having zero baggage allowance possible at all?

Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js]: > would be nice to add to some unit tests for this, covering at least these two cases - 1) you have baggage 2) you don't have any baggage

Good idea!

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js (hunk 1 above)
Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js]: What do you mean? Sorry, I don't understand this one.

Comment (jaroslav-kubicek) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js]: `BaggageDescription` => `BaggageDescription_baggage`

Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js]: `GraphQLCompilerContext: Unknown document `BaggageDescription`.`

File: src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js (hunk 3 above)
Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BaggageInfo/index.js]: :heavy_check_mark: tests added

File: src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js [hunk 4]
```diff
@@ -42,8 +42,7 @@ const styles = css`
     display: inline-block;
     font-size: 22px;
     font-weight: 500;
-    line-height: 1.8;
-    vertical-align: bottom;
+    line-height: 1.2;
```
Comment (jalbertsr) [inline src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js]: Do the same for `BaggageInfo/index.js` line 39

File: src/SingleBookingPage/BookingDetail.js [hunk 5]
```diff
@@ -164,14 +165,16 @@ class BookingDetail extends React.Component<Props> {
...
-          category="baggage"
+        {features.baggage_info && (
+          <FAQExtraInfoButton category="baggage">
+            <BaggageChecked customColor="#00a991" />
+            <p className="iconLabel">Baggage</p>
+          </FAQExtraInfoButton>
+        )}
+        <FAQExtraInfoButton
+          category="boarding-passes"
+          categoryId="RkFRQ2F0ZWdvcnk6ODQ="
```
Comment (jalbertsr) [inline src/SingleBookingPage/BookingDetail.js]: `FAQExtraInfoButton` doesn't need `categoryId` as a prop.

Comment (kantord) [reply src/SingleBookingPage/BookingDetail.js]: Done

File: src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js (hunk 4 above)
Comment (kantord) [reply src/StaticFAQ/FAQExtraInfo/BoardingPassesInfo/index.js]: Done
